### ISSUE TITLE
Can't output to a single file

### ISSUE THREAD

**Issue by nPHYN1T3** [rhasspy/piper#64]
When I follow the example in the docs "cat "sample.txt" | ./piper  --model en-us-amy-low.onnx --output_file sample.wav" sample.wav is constantly overwritten rather than appended to.

***Did a test, seems new lines trigger new files negating the ability to output to a single file.

**Comment by synesthesiam** [comment]
This is something I can add as an option. For now, will this work?

```sh
cat "sample.txt" | xargs | ./piper --model en-us-amy-low.onnx --output_file sample.wav
```

**Comment by nPHYN1T3** [comment]
Yeah I wrote a little front end to cheese it sorta like that. I just figured since there is an output_file option that meant well...output to that ONE file heh. Which I guess technically it does but not in a useful way heh.

**Comment by synesthesiam** [comment]
This will be fixed in the next version (0.1.0). If `--output_file` is given, all of the text input will be combined into a single file.
